modules/objc/examples/code_snippets/SampleCodeTest.m

```diff
@@ -21,32 +21,50 @@
 #import <CouchbaseLite/CouchbaseLite.h>
 #import <CoreML/CoreML.h>
 
+
+/*
+ FIXME: For consistency:
+ 
+ 1. We will keep the '*' with instance after the space.
+    `NSDictionary *dict = [NSDictionary dictionary];`
+ 
+ 2. For making more space in single line, we will avoid space after the ':' in function call.
+    `[myMLModel predictImage:imageData];`
+ 
+ 3. Will only keep `self.database` when using database, query and replicator related code snippet.
+    Except when creating a new database sample.
+ 
+ 4. Will only keep `self.otherDB` when using listener-db
+ 
+ 5. While using replicator/listener, we will use as ivar `self.replicator` and `self.listener` resp.
+ */
+
```

Comment:
As suggested, going forward we could keep these notes as guidelines. I _hope_ have updated the current code base with respect to these notes.

Please feel free to discuss about this, if it's unclear.